packages/statemanager/src/merkleStateManager.ts

```diff
@@ -59,6 +59,7 @@ export const CODEHASH_PREFIX = utf8ToBytes('c')
  * for many basic use cases.
  */
 export class MerkleStateManager implements StateManagerInterface {
+  name = 'MERKLE'
```

Comment:
I don't think you can set this on the interface, but I think for the actual classes, these properties should be marked `readonly` (to also let end-users know it should not be written to/changed)